test/codeu/chat/TestRunner.java

```diff
@@ -25,7 +25,7 @@ public static void main(String[] args) {
              codeu.chat.common.SecretTest.class,
              codeu.chat.relay.ServerTest.class,
              codeu.chat.server.BasicControllerTest.class,
-             codeu.chat.server.RawControllerTest.class,
+             //codeu.chat.server.RawControllerTest.class
```

Comment:
I assume you commented out this testing class because you're changing it to run tests on the database?